src/components/Animal/Animal.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+
+export default function Spot({ spots, animal, organizers }) {
+	return (
+		<article>
+			<header>
+				<h2>{animal.name}</h2>
+			</header>
+
+			<section>
```

Comment:
You could think about outsourcing your three sections to different components in order to make your Animal.jsx more readable. But you don't have to write a story for this kind of refactoring, but can create an issue on your project board where you can link the PR for these changes.

Reply:
I will keep this in mind and think about it as soon as I am working with the 'real' data, because there will be less logic needed then.